packages/arb-token-bridge-ui/src/components/syncers/RetryableTxnsIncluder.tsx

```diff
@@ -48,7 +49,9 @@ export function RetryableTxnsIncluder(): JSX.Element {
       }
 
       // Non-classic - Eth deposit
-      if (isEthDeposit) {
+      // depositTo creates a retryable ticket and acts like a token
+      // when sending ETH to a different address
+      if (isEthDeposit && !isRetryableTicket) {
```

Comment:
We can't use `fetchAndUpdateEthDepositMessageStatus` because ETH transfers to a custom address are technically not ETH transfers, so won't be picked up by this method. Might want to use a better nomenclature for `isRetryableTicket` to be clearer about the intentions of this boolean, but not sure.